CONTRIBUTING.md

```diff
@@ -1,127 +1,168 @@
-sections and are able to sign our CLA.
+This guide describes how to contribute to the Ground open-source project. Please read it in its entirety before you begin.
 
-## Contributor License Agreement (CLA)
+First-time setup:
 
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
+1. Read the [Code of Conduct](#code-of-conduct)
+2. Read and accept the [Contributor License Agreement](#contributor-license-agreement-cla)
+3. [Clone the repo](#cloning-the-repo)
+4. [Set up your environment](#seting-up-your-environment)
+5. [Build and run the app](#building-and-running-the-app)
 
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
+Developing Ground:
 
-## Code reviews
+1. [Claim an issue](#claiming-an-issue)
+2. [Create a new branch](#creating-a-branch)
+3. Iterate on your change
+4. [Create a pull request](#creating-a-pull-request)
 
-Before starting work on a change, comment on one of the [open issues](https://github.com/google/ground-platform/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen)
-saying you'd like to take it on. If one does not exist, you can also
-[create one here](https://github.com/google/ground-platform/issues/new).
+**Note**: This guide assumes you are comfortable working with Git and GitHub. If you're new to Git and/or GitHub, check out [Quickstart - GitHub Docs](https://docs.github.com/en/get-started/quickstart) to help you get started.
 
-Be sure to reference this issue number in your pull request (e.g., 
-"Fix truncated text values #27"), and preferably in your commit message.
+## First time setup
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
-[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
+### Code of Conduct
 
-## Community Guidelines
+Help us keep the Ground project open and inclusive. Please read and follow [Google's Open Source Community Guidelines](https://opensource.google.com/conduct/) when contributing.
 
-This project follows
-[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+### Contributor License Agreement (CLA)
 
-## Setup
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution; this simply gives us permission to use and redistribute your contributions as part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
```

Comment:
nit: s/this simply gives us permission.../this agreement gives us permission...

Reply:
This was copied from other Google open source projects, but I agree this wording it better.